reference/src/util/src/log.c

```diff
@@ -25,7 +25,7 @@ static const char* LOG_LEVEL_STRING[] = {
         "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
 
 #ifndef SA_LOG_LEVEL
-#define SA_LOG_LEVEL LOG_LEVEL_ERROR
+#define SA_LOG_LEVEL LOG_LEVEL_INFO
```

Comment:
Should the default log level be set to info?  Just want to double check this is intentional.

Reply:
This is intentional. The WARN is not currently used and INFO is used in two places.